packages/teams-js/src/public/app.ts

```diff
@@ -424,7 +497,7 @@ export namespace app {
   }
 
   /**
-   * Gets the Frame Context that the App is running in. {@see FrameContexts} for the list of possible values.
+   * Gets the Frame Context that the App is running in. {@link FrameContexts} for the list of possible values.
```

Comment:
Gets the Frame Context that the App is running in. **See** 
{@link FrameContexts} for the list of possible values.

Reply:
Can you add the word "See" before "{@link FrameContexts}" please?